SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/WorkerCoordinator/ManagedWorkerCoordinator.cs

```diff
@@ -201,20 +197,5 @@ private static string GetWorkerFlagOrDefault(Connection connection, string flagN
 
             return defaultValue;
         }
-
-        private void WaitForTargetDeploymentReady(Connection connection)
```

Comment:
Wonder if we should leave this in and instead mark it with a `TODO: UNR-????` to re-enable it's usage when we get DynamicWorkerFlags back